src/core/server/ui_settings/settings/theme.test.ts

```diff
@@ -58,13 +58,10 @@ describe('theme settings', () => {
     const validate = getValidationFn(themeSettings['theme:version']);
 
     it('should only accept valid values', () => {
```

Comment:
I think we should still have tests for v7 and v8 to make sure the changes are backwards compatible.